apps/src/templates/instructions/NonMarkdownInstructions.jsx

```diff
@@ -0,0 +1,33 @@
+var styles = {
+  main: {
+    marginBottom: 35,
+    marginLeft: 80
+  }
+};
+
+var NonMarkdownInstructions = React.createClass({
```

Comment:
I think you're right. Both of these started out more complex, and then got simpler :) In simple forms, I think they fit the bill.